### PR TITLE
[FSPE-6599] Added flex-basis to Flex utilities

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSPE-6599

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://bea9143a7bdca3e3959ecceb7f7a1857b2ccfe40-chi-pr.enterprise.ctl.io" target="_blank"> https://bea9143a7bdca3e3959ecceb7f7a1857b2ccfe40-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1635/ for more details.

File: src/website/views/utilities/flex.pug [hunk 1]
```diff
@@ -414,3 +414,82 @@ ul.-mb--3
       code= '.-flex-' + size + '--shrink0'
     li
       code= '.-flex-' + size + '--shrink1'
+h2 Basis
+p.-text Use the <code>.-flex--basis*</code> class to specifies the initial size of the flex item, before any available space is distributed according to the flex factors.
```
Comment (mattnickles) [inline src/website/views/utilities/flex.pug:418]:
```suggestion
p.-text Use the <code>.-flex--basis*</code> class to specify the initial size of the flex item, before any available space is distributed according to the flex factors.
```

**Review by mattnickles — CHANGES_REQUESTED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1641/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://159e96417c37bcf20f3b3c277894bad73dff362d-chi-pr.enterprise.ctl.io" target="_blank"> https://159e96417c37bcf20f3b3c277894bad73dff362d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1645/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://91fdeac60fe2ce690babffe4fef185f62a981f6d-chi-pr.enterprise.ctl.io" target="_blank"> https://91fdeac60fe2ce690babffe4fef185f62a981f6d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1648/ for more details.

**Review by mattnickles — APPROVED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-gwge3y, https://ops.elasticbox.com/services/instances/i-pep4fo, https://ops.elasticbox.com/services/instances/i-qsl0w5

**Pull request merged**